internal/seclang/directives.go

```diff
@@ -992,6 +992,15 @@ func directiveSecDataset(options *DirectiveOptions) error {
 	return nil
 }
 
+func directiveSecArgumentsLimit(options *DirectiveOptions) error {
+	limit, err := strconv.Atoi(options.Opts)
+	if err != nil {
+		return err
+	}
+	options.WAF.ArgumentLimit = limit
```

Comment:
Since `limit = 0` means no limit is set we should validate this does not assign a `0`